### PR TITLE
Identify as a cooperative

### PR DESCRIPTION
**Pull request by whit537**
This is my proposal to ~~close~~ *move towards* https://github.com/gratipay/inside.gratipay.com/issues/72, by adopting the internationally recognized cooperative values and principles, which [date back to 1844](https://en.wikipedia.org/wiki/Rochdale_Principles). Additionally, this PR solidifies our identity as an *open* cooperative by adopting the values articulated in the recently published [Open Organization Definition](https://opensource.com/open-organization/resources/open-org-definition).

(With these burgeoning values it seemed time to give them their own page on the site; it was always a bit idiosyncratic to have them under "Brand," anyway).

My proposal is that we identify as a _worker_ cooperative, and that adoption of this proposal be by consensus of the nine current members of the Gratipay [project](https://gratipay.com/Gratipay/distributing/) on Gratipay—@whit537 @clone1018 @aandis @chrisdev @sseerrggii @andreottidavide @JessaWitzel @rohitpaulk @mattbk—plus @dmk246 @nobodxbodon and @kaguillera, who would be listed there if https://github.com/gratipay/gratipay.com/issues/4299 were implemented.

Worker cooperatives can have [various structures](http://cultivate.coop/wiki/Worker_co-operative_structures) ranging from informal to formal, and I'm not proposing any change to the structure of our organization at this point, either legal or extra-legal. All I'm proposing is that we identify as an open worker cooperative by adopting the values of a cooperative and an open organization. This identity and these values and principles will guide us as we evolve our organizational structure.

What say you, O potential cofounders of Gratipay-the-open-worker-cooperative!?

-----------------------

# The Seven Cooperative Principles

1. **Voluntary and open membership.** Co-operatives are voluntary organisations, open to all persons able to use their services and willing to accept the responsibilities of membership, without gender, social, racial, political or religious discrimination.
2. **Democratic member control.** Co-operatives are democratic organisations controlled by their members, who actively participate in setting their policies and making decisions. Men and women serving as elected representatives are accountable to the membership. In primary co-operatives members have equal voting rights (one member, one vote) and co-operatives at other levels are also organised in a democratic manner.
3. **Member economic participation.** Members contribute equitably to, and democratically control, the capital of their co-operative. At least part of that capital is usually the common property of the co-operative. Members usually receive limited compensation, if any, on capital subscribed as a condition of membership. Members allocate surpluses for any or all of the following purposes: developing their co-operative, possibly by setting up reserves, part of which at least would be indivisible; benefiting members in proportion to their transactions with the co-operative; and supporting other activities approved by the membership.
4. **Autonomy and independence.** Co-operatives are autonomous, self-help organisations controlled by their members. If they enter into agreements with other organisations, including governments, or raise capital from external sources, they do so on terms that ensure democratic control by their members and maintain their co-operative autonomy.
5. **Education, training and information.** Co-operatives provide education and training for their members, elected representatives, managers, and employees so they can contribute effectively to the development of their co-operatives. They inform the general public - particularly young people and opinion leaders - about the nature and benefits of co-operation.
6. **Cooperation among cooperatives.** Co-operatives serve their members most effectively and strengthen the co-operative movement by working together through local, national, regional and international structures.
7. **Concern for community.** Co-operatives work for the sustainable development of their communities through policies approved by their members.

### CODE REVIEW

File: www/big-picture/values.spt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+*[from the International Co-operative Alliance](http://ica.coop/en/what-co-operative)*
+
+Cooperatives are based on the values of:
+
+- self-help,
+- self-responsibility,
+- democracy,
+- equality,
+- equity, and
```
Comment (nobodxbodon) [inline www/big-picture/values.spt]: nit: remove the ",", "and" to keep the format of lists consistent (with ladder of love part)

Comment (whit537) [inline www/big-picture/values.spt]: Good with 57a1a5410d00c41611729f3d5fd620731f530895?

**Comment by whit537** [comment]
> My only concern is gratipay/gratipay.com#4299, which may not have effect in short term, but under coop principles does become a pre-condition to participate in internal decision making.

Fair point. Should we block this on that?

**Comment by nobodxbodon** [comment]
> Fair point. Should we block this on that?

Practically, I guess it depends on how/if we decide to change internal/external procedures after this, which has been my main question. Nevertheless, IMO collaborator's identities in the community is important, and there are related questions in procedures like how we decide new member. They may be better to be addressed before we settle the new values into stone.

BTW I'm still not clear about the definition of 'membership' in co-op.

**Comment by whit537** [comment]
> BTW I'm still not clear about the definition of 'membership' in co-op.

The definition for us would be "members of the Gratipay project on Gratipay," which a) does seem to suggest that https://github.com/gratipay/gratipay.com/issues/4299 should be a blocker, and b) begs the question, whom do we add as a member of the Gratipay project on Gratipay? ;-)

The answer, of course, is anyone who wants to be added and [behaves well](http://inside.gratipay.com/howto/behave-well). The first cooperative principle is voluntary and open membership!

**Comment by nobodxbodon** [comment]
The definition of membership above is clear enough for now, but seems other co-op have different approaches for member enrolling, like ["pay £1 to become member"](http://www.coop.co.uk/membership/). So does that mean in co-op's definition, a much larger group should be considered to be member? Like those who give to gratipay?

**Comment by andreottidavide** [comment]
> The definition of membership above is clear enough for now, but seems other co-op have different approaches for member enrolling, like "pay £1 to become member". So does that mean in co-op's definition, a much larger group should be considered to be member? Like those who give to gratipay?

I don't think that the members of a cooperative that requires them to pay in order to be enrolled can be compared to the people donating on Gratipay. The first are more like stakeholders, investors (especially given the benefits of being a member of the co-op you used as the example), whereas the second are more like the buyers of a product or service.
From my reading of it, the example of [www.coop.co.uk](www.coop.co.uk) is of a _cooperative_ rather than of a _work-cooperative_, meaning that their members are allowed not to participate in the work of the cooperative if they so choose: the payment they have to provide is therefore needed in order to prove them to be members. In our case of a work-cooperative, the membership requirement is 'to be active in the development of Gratipay' (in other words, to work).

Making a quick analogy - on which I can expand upon if you want me to - from my experience as vice-president of a small non-profit association here in Italy (not a co-op but I think the example will be fitting): we require our members, in order for them to be enrolled for one year, to pay 15€. We use that money as the primary way to fund our projects and members can either not participate or participate (read: work) in those projects. These people would be the members of the Gratipay team (though rather than paying plus working or not working, Gratipay members are only required to work). In my association we also accept donations, from both members and non-members, in order to help fund our projects: these would be the donors giving to Gratipay and they could be members of it or not.

The point I am trying to make is that the two categories, members and donors, are separate.

**Comment by mattbk** [comment]
This looks good to me. Regarding gratipay/gratipay.com#4299, my specific question was answered there.

**Comment by whit537** [comment]
Yeah, cooperatives fall into [several broad categories](http://cultivate.coop/wiki/Types_of_Cooperatives) depending on who the members are. I'm proposing a worker cooperative for Gratipay because, as @andreottidavide suggests, it seems the most natural fit with our "de facto state of affairs." Furthermore, since we [make it so easy](http://inside.gratipay.com/big-picture/welcome) for anyone to become a worker, it didn't seem necessary to overcomplicate the situation with a [multi-stakeholder cooperative](http://cultivate.coop/wiki/Multi-stakeholder_cooperatives).

File: www/big-picture/values.spt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+[---] text/html via markdown
+
+Gratipay is an
+[open](https://opensource.com/open-organization/resources/open-org-definition)
+[cooperative](http://ica.coop/en/what-co-operative) rooted in the
+[open-source](https://en.wikipedia.org/wiki/Open_source) cultural tradition.
+We subscribe to the values of both cooperatives and open organizations, and
+extend these with our own unique value system called the Ladder of Love.
+Together, these three sets of values articulate our collective self-identity,
+and act as a guide when as an open worker cooperative we hash out
```
Comment (nobodxbodon) [inline www/big-picture/values.spt]: Actually I missed this line when I first reviewed. I imagine there will be people like me missing this line and only captured "Open", "cooperative" "open-source", as they are in first sentence and will be highlighted.

Do we need to more obviously states that we are "worker" cooperative?

**Comment by sseerrggii** [comment]
Great @whit537 I am going to follow all this process because we are also thinking of creating a cooperative for my other project https://github.com/coopdevs ;)

**Comment by whit537** [comment]
@EdOverflow I would welcome your input here as well, as you've been [invited](https://github.com/gratipay/inside.gratipay.com/issues/952#issuecomment-268828516) to join the roster.

**Comment by whit537** [comment]
@kaguillera @dmk246 and I had a conversation about this here at HQ a couple days ago. I'll let them chime in with their own thoughts, but here's what I took away from it.


### What do I hope to get out of Gratipay?

Right now I'm the sole owner of Gratipay, LLC. I've put a lot of effort and a little bit of money into the LLC, and identifying as a cooperative commits us to a path towards me owning less of it. What are my hopes and dreams and expectations around being fairly compensated as a founder? Well, it [turns out](https://development.abolishhumanrentals.org/current-structure/startups/) that "[t]his is the central [problem](https://development.abolishhumanrentals.org/structural-problems/startup-investments/) for worker cooperative startups." ([via](https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-272152259)).

Without having actually digested the above links yet, I would say that I hope to withdraw whatever equity I've actually put into Gratipay (cf. #306), but that depends on catching up on our books, which is [a large and ongoing project](https://github.com/gratipay/finances/issues/3). Then I would hope to take money for a long time after I stop working directly on Gratipay. Basically, I'm expecting return on investment to be twyw, just like the rest of our compensation. When after 10 years of taking without working everyone has forgotten who I am, whoever is in charge at that point can take me aside and say, "Chad, I think it's time to phase out" (or, you know, use whatever twyw mechanisms we adopt at scale).

That said, we are _all_ co-founders, because we're pre-sustainability. Once again, the nice thing about twyw is that it's emergent instead of predetermined. As people start making enough to live on from Gratipay, we'll gradually phase over into "sustainability." It's as if we each have twyw equity vs. the boss-determined equity of early hires in old-model startups.


### What are the legal implications?

To realize our identity as a worker-owned cooperative, we're going to need some other worker-owners besides myself. However, we don't want to be so legally rigid that we lose the sense of hospitality that makes Gratipay a special community in the first place. As Miriam Cherry [suggests](https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-274826062), we need to "work around" the lack of "tailor-made enabling statutes" by modifying "already-existing legal structures" to suit our needs.

I've shared some thoughts on legal structure at https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-274826062. However, in order to keep this PR manageable, I still think we should punt on legal and operational specifics and focus here on adopting the cooperative principles.


### How does this help us competitively?

From the in-progress [_Gratipay Guide to to Launching an Open Organization_](https://the.gratipay.guide/):

> _Open_ organizations are [&hellip;] subject to the same market and regulatory forces as any other organization. We have to tell a better story about a better product for a better price with better accountability in order to succeed[.]

We have to compete on product and marketing just like anyone else. All things being equal, however, I would expect our cooperative identity to give us a slight advantage in the open source world, because of the better philosophical alignment with open source.

It could be that being an _open_ cooperative makes us more competitive because we can recruit talent better.

**Comment by whit537** [comment]
> Should we block this on that?

After letting this marinate for a while, I'm ready to proceed. I haven't heard anyone object to this course, and the reality is that even after accepting this PR, I will still be the sole member of the cooperative. This simply expresses Gratipay's intention to be governed by these principles. I see the next step as bringing on a second owner (https://github.com/gratipay/inside.gratipay.com/issues/196), and then broadening the process once we've done it once.

> Do we need to more obviously state that we are "worker" cooperative?

I think it'd actually be better to dial back to just a generic "cooperative," because I think it's too early to commit ourselves to a certain stakeholder class. We may end up multi-stakeholder after all.

Last call for objections!

**Comment by whit537** [comment]
Congratulations, @Gratipay! We are now identified with the cooperative movement through the internationally recognized cooperative values and principles! Our next step is to bring on a second member (#196) with a cooperative member agreement (#72). Based on that we can bring additional members on board.

Huzzah! 🎉  💃  🌻  😄

**Comment by whit537** [comment]
https://twitter.com/Gratipay/status/842483128333680640

**Pull request merged**